### PR TITLE
New gradle

### PR DESCRIPTION
**Pull request by mario**
This would hopefully fix the build for the Analytics branch.

### CODE REVIEW

File: android/gradle/build.gradle [hunk 1]
```diff
@@ -24,41 +25,159 @@ apply plugin: 'findbugs'
...
+        }
+        classpath = files()
+    }
+    check.dependsOn 'checkstyle', 'findbugs', 'pmd', 'lint'
+
+}
+
 dependencies {
     /// dependencies for app building
+    compile name: 'touch-image-view'
```
Comment (MorrisJobke) [inline android/gradle/build.gradle]: I think this caused some issues, but lets see.

**Comment by MorrisJobke** [comment]
To trigger the android container to build you need to create a git tag that starts with `android-` and ends in a number. So I added `android-15` to your commit and we will see if it works:

https://hub.docker.com/r/nextcloudci/android/builds/

File: android/gradle/build.gradle (hunk 1 above)
Comment (MorrisJobke) [inline android/gradle/build.gradle]: Yep

```
> Could not resolve all dependencies for configuration ':_customDebugApkCopy'.
   > Could not find :touch-image-view:.
     Required by:
         :gradle:unspecified
```

**Comment by MorrisJobke** [comment]
For https://github.com/nextcloud/android/pull/679

**Comment by tobiasKaminsky** [comment]
Outdated by #51

**Pull request closed without merging**